_schema.graphql

```diff
@@ -12935,11 +12945,15 @@ type ViewingRoom {
 
   # ID of the partner associated with this viewing room
   partnerID: String!
+  published: Boolean!
```

Comment:
@oxaudo and @mdole I wanted to double-check that this was OK to add this to the Metaphysics schema.